docs/troubleshooting-oauth2.md

```diff
@@ -23,15 +23,15 @@ limitations under the License.
 
 ## Overview {#overview}
 
-This guide covers the most common errors encountered using OAuth 2.0 authentication protocol in OAuth 2.0 authentication backend and management plugin and how to diagnose them.
+This guide covers the most common errors encountered using [OAuth 2.0](./oauth2/) and management plugin and how to diagnose them.
 
 ## Troubleshooting OAuth 2 in the management UI {#management-ui}
 
 ### OpenId Discovery endpoint not reachable {#openid-discovery-endpoint-not-reachable-error}
 
 #### Steps to reproduce the issue
 
-Open the root URL of the management UI in the browser, not authenticated yet. 
+Open the root URL of the management UI in the browser, not authenticated yet.
 Rather than getting the button "Click here to login" you see the following error message:
```

Comment:
what does "not authenticated yet" , what is not authenticated yet, the root url? please make clearer

Reply:
I am removing "not authenticated yet". It is assumed this is the first time the user is trying to login.